Haxe/Static/uhx/compiletime/types/TypeConv.hx

```diff
@@ -1673,6 +1702,29 @@ class TypeConv {
         haxeToGlueExpr:'uhx.internal.HaxeHelpers.dynamicToPointer( % )',
         glueToHaxeExpr:'(uhx.internal.HaxeHelpers.pointerToDynamic( % ) : String)',
       },
+      // cpp.ConstCharStar
```

Comment:
These types are needed now because we are using TypeConv to get the C++ representation of the types in our `@:uexpose` classes. They normally are only used as glue types, so that's why we never needed to define them in here before